.gitignore

```diff
@@ -21,3 +21,4 @@
 setup.sh
 .python-version
 .*.swp
+venv/
```

Comment:
I'm fine with adding `/venv/` to the ignore, but please move it near the other virtual environment dirs (`/env/`, `/env3/`), and ensure there's a newline at the end of the file.